client/src/pages/locations/Form.js

```diff
@@ -84,6 +84,21 @@ const LocationForm = ({ edit, title, initialValues }) => {
     }
   }
 
+  // Location types to be shown to admins in the new location page.
+  const locationTypesAdmin = [
```

Comment:
This can be moved outside the components and the name can be changed to LOCATION_TYPES_ADMIN.